Products/Zing/tests/test_datamaps.py

```diff
@@ -187,6 +177,7 @@ def test_generate_facts_handles_missing_parent_uuid(t, subscribers):
             components.append(
                 Mock(getUUID=Mock(return_value=uuid), getComponentGroupNames=Mock(return_value='getComponentGroupNames')))
```

Comment:
making the components have the same uuid as the device makes this test invalid, as you can't tell if we recorded the correct uuid or not.  having all components return the same set of component groups also makes this invalid as we can't tell if we are correctly recording the component groups for individual components.